src/libs/AxelarQueryAPI.ts

```diff
@@ -278,14 +279,16 @@ export class AxelarQueryAPI {
       throw new Error("Failed to estimate gas fee");
     }
 
-    const destGasFeeWei = parseUnits(
-      (gasLimit * Number(destToken.gas_price)).toFixed(destToken.decimals),
+    const destGasFeeWei = BigNumberUtils.multiplyToGetWei(
```

Comment:
I found wrapping it in the utils class is a bit more confusing, is it better to use it directly here, even it'll be duplicated below?